### PR TITLE
app-shells/atuin: Bump to 18.0.1

### PR DESCRIPTION
**Pull request by freijon**
Simple bump to 18.0.1
Notable changes: Upstream removed the docs from the repository

### CODE REVIEW

File: app-shells/atuin/atuin-18.0.1.ebuild [hunk 1]
```diff
@@ -453,7 +465,6 @@ DOCS=(
 	CONTRIBUTING.md
 	CONTRIBUTORS
 	README.md
-	docs/docs
```
Comment (Flowdalic) [inline app-shells/atuin/atuin-18.0.1.ebuild:456]: Why is this dropped?

File: app-shells/atuin/atuin-18.0.1.ebuild [hunk 2]
```diff
@@ -498,8 +509,6 @@ src_install() {
 
 	if use server; then
 		systemd_dounit "${FILESDIR}/atuin.service"
-	else
-		rm -r "docs/docs/self-hosting" || die
```
Comment (Flowdalic) [inline app-shells/atuin/atuin-18.0.1.ebuild:502]: Why is this dropped?

**Review by Flowdalic — COMMENTED** [review]
Thanks for your contribution.

You dropped a few lines from the latest ebuild, and I am not sure if this was by accident or not?

File: app-shells/atuin/atuin-18.0.1.ebuild (hunk 2 above)
Comment (freijon) [reply app-shells/atuin/atuin-18.0.1.ebuild:502]: As I tried to explain in the PR description, upstream [removed the docs](https://github.com/atuinsh/atuin/pull/1581) from the repository. If you want to keep the docs we would have to source the separate docs repository.

Comment (Flowdalic) [reply app-shells/atuin/atuin-18.0.1.ebuild:502]: I was looking mostly a the commit message, and did not notice it in the github description. The commit message would be the preferred place for this anyways. So in the future, please add such things to the commit message.

Comment (Flowdalic) [reply app-shells/atuin/atuin-18.0.1.ebuild:502]: Meh, it's a pity that they moved the docs into an extra [repo](https://github.com/atuinsh/docs). That makes it only harder for us to ship the documentation as part of the Gentoo package.

**Pull request closed without merging**